app/src/main/java/bitcoin/wallet/core/managers/ExchangeRateManager.kt

```diff
@@ -15,13 +16,13 @@ import java.text.DecimalFormat
 import java.util.*
 import java.util.concurrent.TimeUnit
 
-class ExchangeRateManager: IExchangeRateManager {
+class ExchangeRateManager(currencyManager: ICurrencyManager): IExchangeRateManager {
 
     private var disposables: CompositeDisposable = CompositeDisposable()
 
     init {
-        fetchRatesWithInterval(App.localStorage.baseCurrency)
-        val baseCurrencyDisposable = App.currencyManager.getBaseCurrencyFlowable().subscribe { baseCurrency ->
+        fetchRatesWithInterval(App.currencyManager.baseCurrency)
```

Comment:
What's the difference between App.currencyManager and currencyManager? Can currencyManager be used instead of App.currencyManager?

Reply:
Fixed